prow/cmd/tackle/main.go

```diff
@@ -484,9 +593,7 @@ func applyStarter(kc *kubernetes.Clientset, ns, choice, ctx string, overwrite bo
 	case err != nil: // unexpected error
 		return fmt.Errorf("get plank: %v", err)
 	case !overwrite: // already a plank, confirm overwrite
-		fmt.Printf("Prow is already deployed to %s in %s, overwrite? [no]: ", ns, ctx)
-		var choice string
-		fmt.Scanln(&choice)
+		choice = prompt(fmt.Sprintf("Prow is already deployed to %s in %s", ns, ctx), "no")
```

Comment:
This prompt doesn't make much sense without "overwrite?"

Reply:
Agreed, I am not certain how I lost those extra characters in the conversion.